src/ActionModal/components/SendModal.vue

```diff
@@ -23,7 +23,7 @@
         v-model.number="$v.address.$model"
         v-focus
         type="text"
-        placeholder="Address"
+        placeholder="Address..."
```

Comment:
?

Reply:
This is what Charlie's flow showed :laughing:

Reply:
ah ok. sorry about that. let's remove it for now since it would be inconsistent with all our forms.